core/azure-core/src/main/java/com/azure/core/polling/PollResponse.java

```diff
@@ -0,0 +1,125 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.polling;
+
+import java.time.Duration;
+import java.util.Map;
+
+/**
+ * The life cycle of an operation depicted below
+ * 1. NOT-STARTED
+ * 2. IN-PROGRESS
+ * 3. a. Successfully Complete
+ *    b. User Cancelled
+ *    c. Failed
+ */
+public final class PollResponse<T> {
+
+    private OperationStatus status;
+    private T result;
+    private Duration retryAfter;
+    private Map<Object, Object> properties;
+
+    public enum OperationStatus {
+        NOT_STARTED,
+        IN_PROGRESS,
+        SUCCESSFULLY_COMPLETED,
+        FAILED,
+        USER_CANCELLED
+    }
+
+    /**
+     * Constructor
+     *
+     * @param status : operation status
+     * @param result : the result
+     * @param retryAfter : How long before next retry.
+     * @param properties : the map of perperties
+     */
+    public PollResponse(OperationStatus status,
```

Comment:
Same question about single line arguments

Reply:
I will put constructor arg in single